apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -90,36 +157,115 @@ const SidebarProps = Vue.extend({
   components: {
     LinkItem,
     FileItem,
+    ProfileItem,
     AboutModal,
     HelpModal
   }
 })
 export default class Sidebar extends SidebarProps {
-  //selects/deselects all files. Will select all unless all are already selected
-  toggle_all(): void {
-    if (this.all_toggled) {
-      FilteredDataModule.set_toggled_files([]);
+  // used to toggle v-exapansion-panel
+  file_views = 0;
```

Comment:
This should be an integer as it represents the position of a v-expansion-panel, which is zero-based.  Right now, we only have two expansion panels so 0 and 1 are used.  The number will increase as/if we add more panels.

Maybe an idea to help avoid confusion and increase readability is to do an enumeration.  So, instead of, for instance, 
```file_views = 0```
we could have something like
```file_views = RESULTS_PANEL```

Thoughts?